client/app/src/main/java/com/example/criengine/Adapters/NotificationAdapter.java

```diff
@@ -0,0 +1,80 @@
+package com.example.criengine.Adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.example.criengine.Objects.Notification;
+import com.example.criengine.Objects.Profile;
+import com.example.criengine.R;
+import java.util.ArrayList;
+
+/*
+ * Custom ArrayAdapter that can be used to show notification instances in a ListView
+ * @version 1.0
+ */
+public class NotificationAdapter extends ArrayAdapter<Notification> {
```

Comment:
We should get junit tests for these java code. Could we do that asap? Since it's needed for part 3

Reply:
Oh boy. We got a lot of tests to write